testing/scenario_app/ios/Scenarios/ScenariosUITests/SpawnEngineTest.m

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "SpawnEngineTest.h"
+#import "GoldenImage.h"
+
```

Comment:
might be simpler if you used GoldenTestManager.h

Reply:
I had an eye out for this too.  These tests are kind of a mess.  You don't want to be in a situation where when you add tests you have to edit existing code, you should just be writing new code.  That's why I don't like the GoldenTestManager.  Anything that seemed to have references to platform view tests I tried to avoid, to help us try to get a perspective on what a test should really have.  I didn't have a chance to refactor because my environment is broken.

Reply:
GoldenTestManager isn't related to platform view anymore. It was previously but I refactored it a bit a few weeks ago.

Reply:
This is specifically what I'm talking about: 
 https://github.com/gaaclarke/engine/blob/788f879350fc48199415d0eecf708e33c3d2ebf5/testing/scenario_app/ios/Scenarios/ScenariosUITests/GoldenTestManager.m#L26:L26

This has to be edited to be used.  Once I saw "platform view" I avoided it.